crates/graphql/src/schema/objects/nft.rs

```diff
@@ -158,8 +161,20 @@ impl Nft {
         &self.description
     }
 
-    pub fn image(&self) -> &str {
-        &self.image
+    pub fn image(&self) -> String {
+        let assets_cdn = "https://assets.holaplex.com";
```

Comment:
Configure the asset host in later PR.

Reply:
https://github.com/holaplex/solana-indexer/issues/227